changelog/9.0.4.yaml

```
@@ -0,0 +1,72 @@
+version: 9.0.4
+entries:
+    - kind: other
+      summary: Upgrade Synthetics NodeJS version to latest LTS v20.
+      description: ""
+      component: heartbeat
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/8712
+      issue: []
+      timestamp: 1751029489
+      file:
+        name: 1751029489-synth-upgra-node-v20.yaml
+        checksum: 26a735a19317db577652eea4bb1f3e863212da81
+    - kind: bug-fix
+      summary: Remove incorrect logging that unprivileged installations are in beta.
+      description: 'Unprivileged installations went GA in 8.15.0: https://www.elastic.co/docs/reference/fleet/elastic-agent-unprivileged'
+      component: elastic-agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/8715
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/8689
+      timestamp: 1751037802
+      file:
+        name: 1751037802-Stop-logging-that-unprivileged-installations-are-in-beta.yaml
+        checksum: 2c2f731f90638eaf61e8ec021ce3c4dd8885ff3a
+    - kind: feature
+      summary: Add file logs only mOTEL kube-stack configuration.
```

```suggestion
      summary: Add file logs only managed OTLP input kube-stack configuration.
```

motel is an internal only name